_includes/header.html

```diff
@@ -37,20 +37,8 @@
 					<div class="module left">
 						<ul class="menu">
 							<li>
-								<a href="{{ baseurl }}./#welcome">Welcome</a>
+								<a href="{{ baseurl }}./#welcome">Home</a>
 							</li>	                    
-							<li>
```

Comment:
why we need to remove these things ?

Reply:
The issue maker asked me to do it

Reply:
The issue maker asked me to do the following changes

Reply:
@samyakbvs Please elaborate. Even I'm not able to understand why you're removing these things?